<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugPin</class>
 <widget class="QWidget" name="DebugPin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sendPinValue">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pinValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pinNumber">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>28</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="IOCombo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="boardLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Carte : </string>
   </property>
  </widget>
  <widget class="QComboBox" name="boardCombo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>28</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pinLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pin : </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
